src/assets/local/server.ts

```
@@ -0,0 +1,342 @@
+    // Prepare request data
+    const body = request.body ? await request.text() : null;
+    const requestData = {
+      url: request.url,
+      method: request.method,
+      headers: Object.fromEntries(request.headers),
+      body,
+    };
+
+    // Send message with code, request data, and secrets
+    worker.postMessage({ code, requestData, secrets });
+  });
```

Biome (2.5.6)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)